src/SWP/Bundle/ContentBundle/Resources/config/doctrine-phpcr/Article.phpcr.yml

```diff
@@ -35,6 +35,9 @@ SWP\Bundle\ContentBundle\Doctrine\ODM\PHPCR\Article:
             nullable: true
         isPublishable:
             type: boolean
+        metadata:
```

Comment:
Is there not an issue here with array to string conversion? Looks like metadata is an array in the class, and configured here as a string